### ISSUE TITLE
read only users don't need "really-real time" as much as contributing authors.  Let's use that to our advantage.

### ISSUE THREAD

**Issue by JohnMcLear** [ether/ep_readonly_guest#1]
Given our current SocketIO 10k msgs/sec constraint it may make sense to find a way to make non-contributing "lurkers" able to receive "batches" of changesets as opposed to "really-real-time"..

The reasoning is essentially if instead of a msg every 10ms to a lurker, you do it every 100ms as a batch of changesets you reduce socketio server load by 10x and the latency is/should be barely noticeable to someone lurking...

Something to consider anyway and might not fall within the remit of this plugin.

**Comment by rhansen** [comment]
This will have to be fixed in Etherpad core. Other changes that would reduce overhead from read-only users:
* Don't show read-only users in the user list.
* Don't create an author ID for read-only users.

**Comment by JohnMcLear** [comment]
Agreed, note that "Don't show read-only users in the user list.", it might be nice to see "X number of watchers"
